packages/create-amplify/src/package-manager-controller/package_manager_controller.ts

```diff
@@ -15,10 +15,10 @@ export abstract class PackageManagerController {
    */
   constructor(
     readonly projectRoot: string,
-    protected executable: string,
-    protected binaryRunner: string,
-    protected initDefault: string[],
-    protected installCommand: string,
+    protected readonly executable: string,
+    protected readonly binaryRunner: string,
+    protected readonly initDefault: string[],
+    protected readonly installCommand: string,
```

Comment:
for the comment https://github.com/aws-amplify/amplify-backend/pull/793#discussion_r1437173240 in a previous PR